index.d.ts

```diff
@@ -0,0 +1,21 @@
+import Koa = require("koa");
```

Comment:
@winfinit, exactly what I thought when I wrote this [type definition](https://github.com/auth0/node-jwks-rsa/pull/9). I first added it as a `devDependency` and then why I wrote a small Koa application with TypeScript to test the definition, but it kept complaining about not being able to find `@types/express-jwt`.

After that I saw that the [publishing page](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) on the TypeScript documentation states that the developer has to:

> Make sure all the declaration packages you depend on are marked appropriately in the **dependencies** section in your package.json.